tools/generate_conda_file.py

```diff
@@ -83,7 +83,7 @@
         "https://github.com/explosion/spacy-models/releases/download/"
         "en_core_web_sm-2.1.0/en_core_web_sm-2.1.0.tar.gz"
     ),
-    "transformers": "transformers==2.1.1",
+    "transformers": "transformers>=2.1.1",
```

Comment:
@todd-cook I've done what you suggested here.  We don't have a requirements.txt. We use this python script to generate conda yaml files.

Reply:
Usually it's a good solution to just pin the version there >= 2.1.1. But for those people who already created the environment using the python script generated yaml file, the transformer package will not get auto-upgraded. At least that's what happened at my end. In that case, people will have no problem with function `WarmupLinearSchedule`. However, for those people created the environment with the latest transformer package, function `WarmupLinearSchedule` won't be there. And they will need to use the new function `linear_schedule_with_warmup` and new parameter names.

Even Huggingface won't support transformers older than 2.1, it doesn't mean we won't need to support our code to compatible with its different version.

Reply:
> @todd-cook I've done what you suggested here. We don't have a requirements.txt. We use this python script to generate conda yaml files.

@hlums Thank you for clarifying; I assumed the Setup instructions were Azure only. I like and use Conda, but it does install everything including the kitchen sink; I plain old requirements.txt for minimal installs and ease of debugging.